app/views/content/early-years-teaching-training.md

```diff
@@ -19,74 +19,56 @@ date: "2021-03-29"
         Salary: "No"
-        Funding available: "Yes"
         Length of training: "Typically 1 academic year"
         Qualification: "Early years teacher status"
-      - "Graduate early years teacher training"
   EY-graduate-employment:
     name: feature_table
     arguments:
-      - Cost: "£7,000"
+      - Cost: "£7000 course fees are covered by the Department for Education and are paid directly to the training provider. You don’t need to do anything to access this funding."
         Salary: "Provided by employer"
-        Funding available: "Yes"
         Length of training: "Typically 1 academic year"
         Qualification: "Early years teacher status"
-      - "Graduate employment-based early years teacher training"
+  EY-undergraduate:
+    name: feature_table
+    arguments:
+      - Cost: "Up to £9,250"
+        Salary: "No"
+        Length of training: "3 or 4 years"
+        Qualification: "Early years teacher status"
   EY-self-funded:
     name: feature_table
     arguments:
       - Cost: "Self-funded"
         Salary: "No"
-        Funding available: "No"
         Length of training: "3 months"
         Qualification: "Early years teacher status"
-      - "Graduate 'assessment only' training"
 ---
 
-As an early years teacher you'll work with young children up to the age of 5. Your input will have a lasting and positive effect on their development and wellbeing.
+The early years are critical in children’s development. As an early years practitioner you’ll work with young children up to the age of 5. You’ll use your knowledge and skills to help young children enjoy high standards of teaching and open their minds to new ideas every day.
 
-The early years are critical in children's development. You’ll use your knowledge and skills to help young children enjoy high standards of teaching and open their minds to new ideas every day.
+You’ll need to do ‘early years initial teacher training’ (EYITT) to achieve ‘early years teacher status’ (EYTS). This will allow you to work in a nursery or early years setting. 
 
-## How to become an early years teacher
+If you want to teach in a school then you will need to achieve qualified teacher status (QTS). [Find out more about training to teach primary and secondary pupils](/train-to-be-a-teacher).
 
-You’ll need to do 'early years initial teacher training' (EYITT) to achieve 'early years teacher status' (EYTS).
+## What you need to become an early years practitioner
 
-EYTS is different from '[qualified teacher status](/what-is-qts)' (QTS), which is the equivalent for teaching children aged 3 to 18.
-
-Your training will enable you to meet the [early years teachers' standards](https://www.gov.uk/government/publications/early-years-teachers-standards).
+To do postgraduate early years teacher training, you’ll need a bachelor’s degree or an equivalent including vocational, professional, and overseas qualifications.
```

Comment:
Don't know if I'm just being a bit picky (probs!) but not sure if this makes sense to me like this, just because further down you talk about undergraduate training? I think if I read this first, I'd just assume all training is postgrad then and not read further? Not sure if this needs some sort of extra subheading/context, if that makes sense?

Reply:
It's a good point. I've added a line to clarify. Do you think this makes it clearer @EllieNodder?